src/PureCloud.Client/Contracts/IIntegrationsApi.cs

```diff
@@ -5,175 +5,430 @@ namespace PureCloud.Client.Contracts;
 
 public interface IIntegrationsApi
 {
+    /// <summary>
+    /// Delete Integration.
+    /// </summary>
     Task<Integration> DeleteIntegrationAsync(string integrationId, CancellationToken cancellationToken = default);
 
-    Task<object> DeleteIntegrationsActionAsync(string actionId, CancellationToken cancellationToken = default);
+    /// <summary>
+    /// Delete Action.
+    /// </summary>
+    Task<bool> DeleteIntegrationsActionAsync(string actionId, CancellationToken cancellationToken = default);
 
-    Task<object> DeleteIntegrationsActionDraftAsync(string actionId, CancellationToken cancellationToken = default);
+    /// <summary>
+    /// Delete ActionDraft.
+    /// </summary>
+    Task<bool> DeleteIntegrationsActionDraftAsync(string actionId, CancellationToken cancellationToken = default);
 
-    Task<object> DeleteIntegrationsCredentialAsync(string credentialId, CancellationToken cancellationToken = default);
+    /// <summary>
+    /// Delete Credential.
+    /// </summary>
```

Comment:
Document the parameters along with the method. Apply the same everywhere in the interface